demisto_sdk/commands/common/content/objects/pack_objects/assets_modeling_rule/assets_modeling_rule.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
```

Comment:
why not to implement it as part of the graph objects? we want to deprecate and remove everything under `pack_objects` folder

cc: @ilaner

Reply:
i did add it to the graph objects as well, i've added it as a content object cuz its used in some commands like unify and create-content-artifacts.

Reply:
double check that `unify` needs it, i don't think it uses it anymore, and `create-content-artifacts` we do not use anymore.

cc: @ilaner